src/PrestaShopBundle/Bridge/AdminController/ControllerConfiguration.php

```diff
@@ -185,15 +195,60 @@ class ControllerConfiguration
     /**
      * @var array
      */
-    public $templatesVars = [];
+    public $templateVars = [];
 
     /**
      * @var array
      */
     public $modals = [];
 
     /**
+     * This parameter is needed by legacy helper shop, we can't remove it.
+     *
      * @var int
      */
-    public $multishopContext = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
+    public $multiShopContext = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
```

Comment:
copied from legacy, but its weird, because this results in value `7`. I suppose it should be one of the values instead of all of them combined :smile: